performance-storage-service/openapi.yaml

```diff
@@ -72,10 +127,22 @@ components:
           description: A scale factor for the size of the database
         terminals:
           type: integer
+          description: The number of cores the test was run with
+        query_mode:
+          default: extended
+          description: The query mode the test was executed with
+          enum:
+            - extended
+            - simple
         transaction_weights:
           type: array
+          description: An array of the different transaction types and their weights
```

Comment:
Will the weight always have the same order so that we only need to store the numbers?

Reply:
I think having the API setup this way adds flexibility in case the weights change